docs/content/en/guide/setup.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Setup
+description: ''
+position: 2
+category: Guide
+---
+
+Check the [Nuxt.js documentation](https://nuxtjs.org/guides/configuration-glossary/configuration-modules) for more information about installing and using modules in Nuxt.js.
+
+> Nuxt.js v2.4.0+ is required, earlier versions are not supported
+
+## Installation
+
+Add `@nuxtjs/sentry` dependency to your project:
+
+<code-group>
+  <code-block label="Yarn" active>
+
+  ```bash
+  yarn add @nuxtjs/sentry
+  ```
+
+  </code-block>
+  <code-block label="NPM">
+
+  ```bash
+  npm install @nuxtjs/sentry
+  ```
+
+  </code-block>
+</code-group>
+
+Then, add `@nuxtjs/sentry` to the `modules` section of `nuxt.config.js`:
+
+```js[nuxt.config.js]
```

Comment:
Please add a space after the initial `js`. Otherwise, it breaks syntax highlighting in some editors.